### PR TITLE
Update to new osl-firewall resources

### PR DESCRIPTION
**Pull request by detjensrobert**

### CODE REVIEW

File: envvars [hunk 1]
```diff
@@ -1,4 +0,0 @@
-cookbook=osl-imap
-version=1.0.0
-envs=~
-pr_link=https://github.com/osuosl-cookbooks/osl-imap/pull/1
```
Comment (ramereth) [inline envvars]: Let's not remove this as I think it may cause an issue.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by ramereth** [comment]
@detjensrobert can you please resolve the failures?

**Review by ramereth — APPROVED** [review]

**Comment by detjensrobert** [comment]
[osl-imap](https://trello.com/c/Dy2PApNi/2114-osl-imap)

**Comment by detjensrobert** [comment]
!bump major

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v2.0.0.  Have a nice day!

**Pull request merged**